src/main/java/core/basesyntax/utils/transaction/handlers/impl/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.utils.transaction.handlers.impl;
+
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.transaction.FruitTransaction;
+import core.basesyntax.utils.transaction.handlers.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    public void perform(FruitTransaction transaction, StorageImpl storage) {
+        storage.updateEntry(transaction.getProduct(), transaction.getQuantity());
+    }
+}
```

Comment:
According to the checklist, the class should not begin with an empty line. Please remove the empty line at the start of the class definition.